grpc/src/main/java/com/linecorp/armeria/server/grpc/UnframedGrpcService.java

```diff
@@ -221,7 +221,7 @@ private void frameAndServe(
             return;
         }
 
-        grpcResponse.aggregate().handleAsync(
+        grpcResponse.aggregate(ctx.eventLoop()).handleAsync(
```

Comment:
What happens if we revert this?

Reply:
I found an async stacktrace while debugging this code.
- An `HttpRequest` is closed by `Http2RequestDecoder.onDataRead()` and the close event directly calls `grpcResponse.aggregate()`.
- `aggregate()` checks a `RequestContext` to acquire an executor. However, The event loop does not have an request context in the thread-local. 
- Because it was called by Netty. So I explicitly specified `ctx.eventLoop()`.

Reply:
I see. No worries, then. :-)

Reply:
How about using `aggregateWithPooledObjects`?